Makefile

```diff
@@ -442,14 +444,20 @@ $(BUILD)/filesystem.bin: \
 	sleep 2
 	pgrep redoxfs-fuse
 	cp -RL filesystem/* $(BUILD)/filesystem/
-	chown -R 0:0 $(BUILD)/filesystem/
-	chown -R 1000:1000 $(BUILD)/filesystem/home/user/
-	chmod 700 $(BUILD)/filesystem/root/
-	chmod 700 $(BUILD)/filesystem/home/user/
+	chmod -R uog+rX $(BUILD)/filesystem
+	chmod -R uog-w $(BUILD)/filesystem
+	chmod -R 555 $(BUILD)/filesystem/bin/
+	chmod -R u+rwX $(BUILD)/filesystem/root
+	chmod -R og-rwx $(BUILD)/filesystem/root
+	chmod -R u+rwX $(BUILD)/filesystem/home/user
+	chmod -R og-rwx $(BUILD)/filesystem/home/user
+	chmod og-r $(BUILD)/filesystem/etc/passwd
```

Comment:
It is /etc/shadow, which is not present on Redox, that should be set to og-r. Setting /etc/passwd is a bad idea, as it is used to grab information about the current user by processes running as the current user